### ISSUE TITLE
bs_sign doesn't cleanup temporary gpg home directory when running in cpio mode

### ISSUE THREAD

**Issue by ismail** [openSUSE/obs-sign#28]
What do we observe:

When bs_sign is signing in cpio mode (e.g. signing kernel-default) a lot of gpg-agent process kept hanging (at some point up to ~1000), this seems to be because after sign is finished the directories under `/var/run/signd/$temp` is not cleaned up, observed with a simple `ls /var/run/signd`. 

This has many interesting effects on the system, as more and more gpg-agent processes are started the inotify process limits (by default 128) is reached and this prevent any new agent working, resulting in logs like:

```
sign failed: openssl sign: gpg: key 63941AC9A5A05EA2: accepted non self-signed user ID "privsign"
gpg: can't connect to the agent: IPC connect call failed
gpg: key 63941AC9A5A05EA2: accepted non self-signed user ID "privsign"
gpg: can't connect to the agent: IPC connect call failed
gpg: error getting the KEK: No agent running
```

**Comment by mlschroe** [comment]
Frank, wouldn't using CLEANUP => 1 in the tempdir call do the trick?

**Comment by ismail** [comment]
Fixed by ef84872c97516881c2d70e5ed09e25d295821f99, thanks a lot. Is it possible to do a new release? @M0ses

**Comment by ismail** [comment]
Fixed in 2.5.9
